client/sites-dashboard/components/scroll-to-top-button.tsx

```diff
@@ -0,0 +1,57 @@
+import { Gridicon } from '@automattic/components';
+import styled from '@emotion/styled';
+import { useCallback, useEffect, useState } from 'react';
+
+type ScrollButtonProps = {
+	classname?: string;
+};
+
+const ScrollButton = styled.button< { visible: boolean } >`
+	position: fixed;
+	display: flex;
+	opacity: ${ ( props ) => ( props.visible ? 1 : 0 ) };
+	align-items: center;
+	justify-content: center;
+	bottom: 40px;
+	left: 40px;
+	z-index: 1;
+	height: 42px;
+	width: 42px;
+	background-color: #000;
+	color: #fff;
+	border-radius: 4px;
+	transition: opacity 0.3s ease-in-out;
+`;
+
+export const ScrollToTopButton = ( { classname }: ScrollButtonProps ) => {
+	const SCROLL_THRESHOLD = 400;
```

Comment:
Any reason for that specific threshold? How about using the window's height instead? We'll need to listen to it.

Reply:
window height seems more reasonable, although I will add a threshold prop that we can provide to use instead of window height if we so choose.